### ISSUE TITLE
[Accessibility] - Use proper anchor tag for project links in Explore page, so we can open it in new tab

### ISSUE THREAD

**Issue by doaortu** [thefullstackgroup/thefullstack#22]
Hi, everyone. Thanks for making this awesome website.

One thing that annoys me when using the Explore page, is that I can't open each project in new tab. Because most of the time, I just need to peek the details of what the project about.

![links](https://github.com/thefullstackgroup/thefullstack/assets/113872927/fc78264f-bbfa-40ed-8f5e-6e3faa8bc216)

It would be very helpful if we can open as new tab for those project links.

What do you guys thinks? Is it doable? Or is there a better alternatives?

Thank you in advance.

PS: I can submit a PR if you think this is a good idea.

**Comment by danoely1979** [comment]
Agree, this is super annoying. We're gonna be making some improvements to the site over the next while. We've been quiet lately due to being super busy with other work commitments.
